### PR TITLE
Add WATM festival schedule

### PR DESCRIPTION
**Pull request by shinedesigns**
I know @howtohumanbetter wants this up ASAP despite the TBD elements... I still want to add color to the schedule section, and improve styling of the event boxes and detail lightboxes. I also want to rework the lightbox partials, but as it is I think this is clear and clean enough to merge.

### CODE REVIEW

File: _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+type: watm-event
+title: connection-practice-meeting
+name: Connection Practice Meeting
+facilitator: Michelle Kelley
```
Comment (j00bar) [inline _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md:5]: Is her name Kelly or Kelley?

File: _sections/events/watm-festival/2020-06-12-2130-dance-like-nobodys-watching.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+date: '2020-06-12'
+start-time: '21:30'
+end-time: '23:30'
+link: TBA
+calendar-link:
+---
+
+Tap into your "inner child". Remember when you were a child, dancing and bouncing around on your parent's bed? This dance party will be broadcast from my bedroom and my bed. I will take musical requests via chat. Let’s dance like nobody's watching.
+
+Cameron Fairfax Payne age: 50, Profession: tie-dye artist for 35 years, Recovery time; 1 year, diagnosed with ADHD at 5 years old. prescribed stimulant therapy until age 20. Was in active addiction from about 13 years old. Seek Healing and the network of facilitators has helped me with direction and support in my healing path.
```
Comment (j00bar) [inline _sections/events/watm-festival/2020-06-12-2130-dance-like-nobodys-watching.md:16]: SeekHealing one word, right?

File: _sections/events/watm-festival/2020-06-14-0900-listening-training-sample-class.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+date: '2020-06-14'
+start-time: '09:00'
+end-time: '11:00'
+link: TBA
+calendar-link:
+---
+
+Listening Training is SeekHealing's "core curriculum" for volunteers that reframes everything you think you know about addiction & socio-emotional wellness.  Whether you’re in early recovery, long-term recovery, or just looking to upgrade your communication skills, Listening Training is designed to support you in having more authentic and fulfilling relationships.
+
+Ionsul is a coach, facilitator, and trainer in the art of building genuine connections and satisfying relationships. She has had her own private practice since 2015, and has facilitated SeekHealing’s core curriculum, Listening Training, since 2019. She is a mother of two and loves spontaneous dance parties in the kitchen. You can reach her at ionsul@seekhealing.org
```
Comment (j00bar) [inline _sections/events/watm-festival/2020-06-14-0900-listening-training-sample-class.md:16]: `mailto:`

File: _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+day: 1
+date: '2020-06-12'
+start-time: '17:00'
+end-time: '18:00'
+link: TBA
+calendar-url:
```
Comment (j00bar) [inline _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md:11]: `calendar-link` right?

File: _sections/events/watm-festival/2020-06-12-1630-opening-ceremony.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+facilitator: Jennifer Nicolaisen
+day: 1
+date: '2020-06-12'
+start-time: '16:30'
+end-time: '16:50'
+link: TBA
+calendar-link:
+---
+
+Join the SeekHealing Executive Director for a brief introduction and opening meditation to welcome yourself to “We Are The Medicine’.
```
[the remark this answers is not in the document]
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-1630-opening-ceremony.md:14]: What do you mean?

File: _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md (hunk 1 above)
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md:5]: Good catch -- looks like it's "Kelly" in Slack

File: _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+One seeker always facilitates the meeting (to guide the conversation, manage time, and introduce different exercises to help the group feel connected), and another always volunteers as Space Owl (to step aside with anyone who has a big emotional experience and needs a 1-on-1 connection).
+
+There is no hierarchy in Connection Practice meetings, and everyone is encouraged to share equally and freely, including the person who facilitates.
+
+Michelle Kelly is passionate about connecting with others. Her whole life she has been a journey through relational adventures and misadventures. Over the past few years she has been able to hone her access to relational tools through personal relationships, as well as various relational training and certification programs. Michelle holds space for connection practices meetings for SeekHealing, and is most excited when everyone finds a way to meet in their heart and the present moment together. She views this experience as transcendent. She sees relating deeply with others as a way to get closer to truth and knowing the self.
```
[the remark this answers is not in the document]
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md:20]: Kelly

File: _sections/events/watm-festival/2020-06-12-2130-dance-like-nobodys-watching.md (hunk 2 above)
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-2130-dance-like-nobodys-watching.md:16]: Eagle Eyes over here! I am but a humble copy/paster, tending to my reams of text.

File: _sections/events/watm-festival/2020-06-14-0900-listening-training-sample-class.md (hunk 3 above)
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-14-0900-listening-training-sample-class.md:16]: Thank you, I had meant to do that

File: _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md (hunk 4 above)
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md:11]: Yes, thank you. I'll make these consistent -- not sure how they ended up not.

File: _sections/events/watm-festival/2020-06-12-1630-opening-ceremony.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (shinedesigns) [reply _sections/events/watm-festival/2020-06-12-1630-opening-ceremony.md:14]: Oh, never mind -- I see it.

**Comment by shinedesigns** [comment]
Everything flagged thus far has been fixed

**Pull request merged**